# goal definition
bata_base/BataTraj traj
---
# result definition
bool[] waypoint_success
---
# feedback
bool[] waypoint_success
